pom.xml

```diff
@@ -161,6 +161,49 @@
             </dependencyManagement>
         </profile>
 
+        <profile>
+            <id>cdh6.3.3</id>
```

Comment:
Can you rename the profile to CDH-6.3? This would be consistent with CDH-5.15

Reply:
ok, done